src/types/item.schema.js

```diff
@@ -2,26 +2,26 @@ const { gql } = require('apollo-server');
 
 const typeDefs = gql`
   type Item {
-  pageid: Int!
-  ns: Int!
-  title: String!
-  lastrevid: Int!
-  modified: String!
-  type: String!
-  id: String!
-  labels: [Label]
-  ## Implement those later
-  # descriptions: [Description]
-  # aliases: [Alias]]
-  # claims: [Claim]
-  # sitelinks: [Sitelink]
+    pageid: Int!
+    ns: Int!
+    title: String!
+    lastrevid: Int!
+    modified: String!
+    type: String!
+    id: String!
+    label(language: String): Label!
```

Comment:
We can't guarantee that there's a label for every language, so this should be nullable.

```suggestion
    label(language: String): Label
```